### PR TITLE
Mila/redesign sidebar

### PR DESCRIPTION
**Pull request by mila-sevilla**

### CODE REVIEW

**Comment by pzhine** [comment]
Some notes
- [ ] Pink highlight in navbar should start active on pages row
- [ ] Entering search mode should update pink highlight to search row
- [ ] Clicking a navbar icon while active should hide the sidebar
- [ ] Pink highlight should go away when sidebar is hidden
- [ ] Clicking a navbar icon when sidebar is hidden should navigate and show the sidebar
- [ ] Clicking "New Page" should update pink highlight to pages row
- [ ] Hamburger icon should align with logo as in mocks
- [ ] Use regular hamburger for both states (as in mocks now)
- [ ] Logo should be updated to plaintext logo as in mocks

**Comment by mila-sevilla** [comment]
@pzhine Should be all implemented now, could you have a look?

File: packages/databyss-ui/components/Sidebar/SidebarIconButton.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  onClick,
+  isActive,
+  borderPosition,
+  title,
+  ...others
+}) => {
+  const { isMenuOpen } = useNavigationContext()
+
+  return (
+    <>
```
[the remark this answers is not in the document]
Comment (mila-sevilla) [reply packages/databyss-ui/components/Sidebar/SidebarIconButton.js:20]: Yes, it's just a shorthand syntax :)

File: packages/databyss-ui/components/Sidebar/SidebarIconButton.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      <View
+        bg="purple.2"
+        position="absolute"
```
[the remark this answers is not in the document]
Comment (mila-sevilla) [reply packages/databyss-ui/components/Sidebar/SidebarIconButton.js:23]: Yes, to achieve this move down/ move up effect of the pink border. I manipulate the position by modifying the "top" value (which only works with position absolute) and with the transition it looks like it's moving up and down.

File: packages/databyss-ui/modules/Sidebar/Sidebar.js [hunk 3]
```diff
@@ -30,72 +27,48 @@ Section.defaultProps = {
...
       <View
-        widthVariant="content"
-        theme={darkTheme}
-        bg="background.0"
-        pt="medium"
-        height="100vh"
+        {...defaultProps}
+        position="relative"
+        css={css({
+          width: sidebarWidth,
```
[the remark this answers is not in the document]
Comment (mila-sevilla) [reply packages/databyss-ui/modules/Sidebar/Sidebar.js:47]: It was used like this before so I assumed you needed to add the css. But I just tried and yeah, it doesn't need it! I removed it now :)

File: packages/databyss-ui/modules/Sidebar/Sidebar.js [hunk 4]
```diff
@@ -30,72 +27,48 @@ Section.defaultProps = {
   variant: 'heading3',
 }
 
+export const sidebarWidth = 240
```
[the remark this answers is not in the document]
Comment (mila-sevilla) [reply packages/databyss-ui/modules/Sidebar/Sidebar.js]: Yes, good idea! Should be done now

**Pull request merged**